App/Screens/Helpers/Header.js

```diff
@@ -1,82 +1,82 @@
-// import React, { Component } from 'react';
+import React, { Component } from 'react';
 
-// import {
-//   StyleSheet,
-//   Text,
-//   View,
-//   Image,
-//   Animated,
-//   ScrollView,
-// } from 'react-native';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  Animated,
+  ScrollView,
+} from 'react-native';
 
-// // import Octocat from "./octocat.png";
+import Octocat from "../images/octocat.png";
 
-// var styles = StyleSheet.create({
-// 	container: {
-// 	    flex: 1
-// 	},
-// 	titleStyle: {
-// 	    marginBottom: 10
-// 	},
-// 	image: {
-// 	    width: 50,
-// 	    height: 50
-// 	},
-// 	header: {
-// 	    paddingTop: 30,
-// 	    alignItems: "center"
-// 	},
-// 	scrollView: {
-// 	    flex: 1
-// 	}
-// });
+var styles = StyleSheet.create({
+	container: {
+	    flex: 1
+	},
+	titleStyle: {
+	    marginBottom: 10
+	},
+	image: {
+	    width: 50,
+	    height: 50
+	},
+	header: {
+	    paddingTop: 30,
+	    alignItems: "center"
+	},
+	scrollView: {
+	    flex: 1
+	}
+});
 
-// class Header extends Component {
-// 	componentWillMount() {
-//     	this.animated = new Animated.Value(0);
-//   	}
+export class Header extends Component {
+	componentWillMount() {
+    	this.animated = new Animated.Value(0);
+  	}
 
-// 	render() {
-// 		const hideImageInterpolate = this.animated.interpolate({
-// 		  inputRange: [0, 250],
-// 		  outputRange: [50, 0],
-// 		  extrapolate: "clamp",
-// 		})
+	render() {
+		const hideImageInterpolate = this.animated.interpolate({
```

Comment:
These seem like they are tabbed too far in. Remind me to make sure your text editor is set up to use two spaces instead of tabs. It makes the formatting on github more consistent.